data-es6.js

```diff
@@ -778,6 +778,29 @@ exports.tests = [
       },
     },
     {
+      name: 'lexical super-constructor binding',
+      exec: function(){/*
+        var received
+
+        class B {
+          constructor (arg) {
+            received = arg
+          }
+        }
+        class C extends B {
+          constructor () {
+            var callSuper = () => super('foo')
+            callSuper()
+          }
+        }
+
+	return new C instanceof C && received === 'foo'
+      */},
+      res: {
+      	/* TBD */
```

Comment:
We should really look into making a script that can connect to multiple browsers, run snippet of code, and report results back. Something like http://tutti.tobyho.com/ (which used to work back in the days). Perhaps using @saucelabs or @browserstack API's?

@jdalton I know you've worked with some of these; do you know if it's possible?